templates/clients.html

```diff
@@ -1,171 +1,157 @@
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                    <td>
-                        {% if client.features.e2ee %}
-                        <span class="checkmark">✓</span>
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                    <td>
-                        {% if client.features.replies %}
-                        <span class="checkmark">✓</span>
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                    <td>
-                        {% if client.features.stickers %}
-                        <span class="checkmark">✓</span>
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                    <td>
-                        {% if client.features.multi_account %}
-                        <span class="checkmark">✓</span>
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                    <td>
-                        {% if client.features.voip %}
-                        <span class="checkmark">✓</span>
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                    <td>
-                        {% if client.features.reactions %}
-                        <span class="checkmark">✓</span>
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                    <td>
-                        {% if client.features.sso %}
-                        <span class="checkmark">✓</span>
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                    <td>
-                        {% if client.features.edits %}
-                        <span class="checkmark">✓</span>
-                        {% else %}
-                        <span class="cross">✗</span>
-                        {% endif %}
-                    </td>
-                </tr>
-                {% endfor %}
-            </tbody>
-        </table>
+
+    <h2>All known clients</h2>
+    <p>
+        The Matrix ecosystem is vibrant: whatever platform you use, there are
+        good chances someone has developed a client for it. Here we list all the
+        clients the Foundation is aware of, with a maturity score and a list of
+        important features they support or not to help you pick the best for
+        you.
+    </p>
```

Comment:
I'm not too happy about this being in the template, but I'm not really sure what's the best place to put it either